Makefile

```diff
@@ -26,3 +26,6 @@ spec:
 
 behat:
 	vendor/bin/behat --colors --strict --stop-on-failure --no-interaction -vvv -f progress
+
+profile:
+	docker-compose exec blackfire blackfire curl -L http://app
```

Comment:
Can `http://app` be replaced with an argument?
```suggestion
	docker-compose exec blackfire blackfire curl -L $(url)
```

Reply:
@coldic3 Sure! I put also the execution example